tests/ha/test_data_pod_failure.py

```diff
@@ -1718,6 +1718,7 @@ def test_mpu_after_unsafe_pod_shutdown(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.skip(reason="Functionality not Available")
+    @pytest.mark.skip(reason="VM issue in after Restart(LRL-3413). Need to be tested on HW")
```

Comment:
why 2 skip markers, first one is good enouh as HA pod failover functionality is not available?